packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -198,23 +186,21 @@ const DatePickerInput = (props) => {
     delete inputAttributes.id;
   }
 
+  let dayInputRef = document.getElementById(dayInputId);
+  let monthInputRef = document.getElementById(monthInputId);
+  let yearInputRef = document.getElementById(yearInputId);
```

Comment:
We would get null input refs if no inputAttributes.id was passed in. If so, wouldn't it throw an error later on when it tries to access these refs?

Reply:
That's correct !! this would be null when `id` is not provided. Actually in DateInput ref's gets assigned on render by the node provided by terra-form-input' refCallback. hence no need of `document.getElementById()` to get element reference. 
Updated [here](https://github.com/cerner/terra-framework/pull/1563/commits/51a58975f288f50212a95e64c66f2e76766e1a92#diff-ed641a9a9e86097e537f02f796210b67af89a5b2eefac5add0745b656086de82)